health/micro-ui/web/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignHome.js

```
@@ -165,14 +167,14 @@ const CampaignHome = () => {
           <div className={"descStyle"}>{t("HCM_IMPORT_EXISTING_CAMPAIGN")}</div>
         </div>
         <div
-          className={"cardStyle disabledCard"}
+          className={"cardStyle"}
           onClick={() => {
-            // Add functionality for campaign templates
+            navigate(`/${window.contextPath}/employee/campaign/campaign-templates`);
           }}
           tabIndex={2}
         >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)